### ISSUE TITLE
[Bug]: Environment variable isn't available on a container from an image builded with that variable

### ISSUE THREAD

**Issue by yunielrc** [yunielrc/vedv#54]
## 🐛 Bug Report

I declared the variable `ENV IMAGE_BUILDING=true` on the Vedvfile, the image 
build succeed, but when I created a container from the image the variable 
 isn't available.

## OS Environment

```sh
    OS: Linux 6.1 Manjaro Linux
    CPU: (12) x64 12th Gen Intel(R) Core(TM) i5-12400F
    Memory: 16.60 GB / 31.20 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
```

**Comment by yunielrc** [comment]
## Bug

The variables aren't exported on file: /etc/vedv-guest/layer/env

## Solution

Add the `export` keyword to variables on the file: /etc/vedv-guest/layer/env

**Comment by yunielrc** [comment]
closed by 204c669
